src/entities/viewer/model/api.ts

```diff
@@ -8,7 +9,65 @@ export const login = () => auth0.loginWithPopup();
 export const logout = () => auth0.logout();
 export const getUser = async (): Promise<User> => {
   const user = await auth0.getUser();
-  if (!user?.nickname || !user?.name) throw new Error('Something went wrong..');
-  return { name: user.name, nickname: user.nickname };
+  if (!user?.nickname || !user?.sub || !user?.email || !user.picture)
+    throw new Error('Something went wrong..');
+
+  return {
+    email: user.email,
+    surname: user?.given_name || ' ',
+    name: user?.given_name || ' ',
+    nickname: user.nickname,
+    user_id: user.sub,
+    picture: user.picture,
+  };
 };
+
 export const getAuthState = (): Promise<boolean> => auth0.isAuthenticated();
+
+export const buildReq = async ({
```

Comment:
buildReq ? what is it? unreadable name

Reply:
No, its better to move this logic away from this place. Use setToken own api on lower layer

Reply:
removed this logic